bosh/ruby-app-release/jobs/rubyweb/templates/ctl.erb

```diff
@@ -20,28 +20,17 @@ case $1 in
     
       start_file = p('bootstrap')
       if not start_file.empty?
-        raise 'No bootsrap file provided'
+        raise 'No bootstrap file provided'
       end
 
-      if (start_file.eql? "app.rb")
-        raise 'Wrong bootstrap file provided'
-      end
-    
     %>
     bundle exec ruby <%= start_file unless start_file.eql? "app.rb" %> \
       >>  $LOG_DIR/rubyweb.stdout.log \
       2>> $LOG_DIR/rubyweb.stderr.log
-
-    ;;
-
-  start)
-
-    rm -f $PIDFILE
-    
     ;;
 
   *)
-    echo "Usage: ctl {start|stop}" ;;
+    echo "Usage: ctl {start|stop}"
```

Comment:
This adds a simple bug for the candidate; this bug is checked by that parser thing I described in [another comment](https://github.com/anynines/homework/pull/9/files#r1023955447)